TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/support/SocketConnectionObservable.java

```diff
@@ -8,9 +8,7 @@
 import io.socket.client.SocketIOException;
 import io.socket.emitter.Emitter;
 
-import static io.socket.client.Socket.EVENT_CONNECT;
-import static io.socket.client.Socket.EVENT_CONNECT_ERROR;
-import static io.socket.client.Socket.EVENT_ERROR;
+import static io.socket.client.Socket.*;
```

Comment:
Now it has four it constantly updates it automatically, I'll see if I can disable later.